ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.ts

```diff
@@ -0,0 +1,31 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: 0x3B8, // @TODO: change to zoneid entry when available
+  triggers: [
+    // Tankbuster shared by 2nd and 3rd bosses
+    {
+      id: 'Isitva Siddhi',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '62A9', source: '(?:Sanduruva|Minduruva)', capture: true }),
```

Comment:
```suggestion
      netRegex: NetRegexes.startsUsing({ id: '62A9', source: ['Sanduruva, 'Minduruva'], capture: true }),
```

Not that it matters *that* much, but it's probably better not to use raw regex syntax within regex helpers.